.qawolf/banks.test.js

```diff
@@ -0,0 +1,45 @@
+const qawolf = require('qawolf')
+const fs = require('fs')
+const path = require('path')
+
+let browser
+let page
+
+beforeAll(async () => {
+  browser = await qawolf.launch()
+  const context = await browser.newContext()
+  await qawolf.register(context)
+  page = await context.newPage()
+})
+
+afterAll(async () => {
+  await qawolf.stopVideos()
+  await browser.close()
+})
+
+const screenshot = (scenarioName, screenshotName) => {
+  const directory = path.join('.qawolf/screenshots/', scenarioName)
+  if (!fs.existsSync(directory)) {
+    fs.mkdirSync(directory, { recursive: true })
+  }
+  return page.screenshot({ path: path.join(directory, `${screenshotName}.png`) })
+}
+
+test('banks transactions', async () => {
+  await page.goto('http://banks.cozy.tools:8080/')
+  await page.click('#password')
+  await page.type('#password', 'cozy')
+  await page.click('text=SE CONNECTER')
```

Comment:
What if we had a data-testid on the button? Does QAWolf use it or use the string?

Reply:
It will use the data-testid.

Reply:
Thanks!